bundles/org.openhab.core.io.rest.core/src/main/java/org/openhab/core/io/rest/core/internal/fileformat/FileFormatResource.java

```diff
@@ -175,6 +181,26 @@ public class FileFormatResource implements RESTResource {
                       param: my param value
             """;
 
+    private static final String DSL_SITEMAPS_EXAMPLE = """
+            sitemap MySitemap label="My Sitemap" {
+                Frame {
+                    Input item=MyItem label="My Input"
+                }
+            }
+            """;
+
+    private static final String YAML_SITEMAPS_EXAMPLE = """
+            version: 2
```

Comment:
version should be 1.

I see it was kept to 2 for other examples in that file, that is to be fixed in a separate PR so that it can be easili backported to 5.0.x. I will create it.